segmenters/nlp/SpacySentencizer/tests/test_sentencizer.py

```diff
@@ -0,0 +1,55 @@
+import pytest
+import spacy
+
+from .. import SpacySentencizer
+
+
+def test_multilingual_sentencizer_with_en_lang():
+    sentencizer = SpacySentencizer()
+    text = "It is a sunny day!!!! When Andy comes back, we are going to the zoo."
+    crafted_chunk_list = sentencizer.segment(text, 0)
+    assert len(crafted_chunk_list) == 2
+
+
+def test_multilingual_sentencizer_with_id_lang():
+    sentencizer = SpacySentencizer()
+    text = "ini adalah sebuah kalimat. ini adalah sebuah kalimat lain."
+    crafted_chunk_list = sentencizer.segment(text, 0)
+    assert len(crafted_chunk_list) == 2
+
+
+def test_sentencier_cn():
+    sentencizer = SpacySentencizer()
+    text = "今天是个大晴天！安迪回来以后，我们准备去动物园。"
+    crafted_chunk_list = sentencizer.segment(text, 0)
+    assert len(crafted_chunk_list) == 1
+
+
+def test_unsupported_lang():
+    dummy1 = spacy.blank("xx")
+    dummy1.to_disk("/tmp/xx")
```

Comment:
if u store things to disk during test time, please make use of tmpdir fixture of pytest